google/ggrc-core
# GGRC-6757 "There was an error" (POST 400), if create custom role with Non-ASCII symbols

# Issue description

"There was an error" (POST 400), if create custom role with Non-ASCII symbols

# Steps to test the changes

Steps to reproduce:

1. Log in GGRC app as admin user
2. Open Admin page > Custom roles tab
3. Open devtools (Network tab) > add custom role to any object with Non-ASCII symbols (ö)

Actual Result: 'There was an error' is displayed in UI, POST 400 in Network tab
Expected Result: No errors should be displayed. The system should handle the case if user creates custom role with Non-ASCII symbols

# Solution description

Top level general exception handling lead to error code being appended to the response to client. Thus even though the custom role was actually created client reacted on error code, showed error widget and don't actually follow expected flow.

# Sanity checklist


# PR Review checklist
